SPECS-EXTENDED/rear/rear.spec

```diff
@@ -91,7 +91,7 @@ Requires: mkisofs
 
 ### On RHEL/Fedora the genisoimage packages provides mkisofs
 %if %{?centos_version:1}%{?fedora:1}%{?rhel_version:1}0
```

Comment:
What is this macro doing on Mariner?  Are the Requires that follow actually being included in the Mariner Build?  Also the %if 0%{?rhel} a few lines down is suspicious.  At a glance I think we want to remove those for Mariner.

Reply:
I think you're right that none of these are being included. However, it seems that the checks for each distro are because everyone adds `mkisofs` from different packages. We're using cdrkit, so should I add a BuildRequires for that when removing the flags?

Reply:
I was able to remove the comments and add `cdrkit` to the Requires, but I saw `lsb` referenced by other distros. I found lsb_release, but I can't find a package that has LSB.